rllib/algorithms/algorithm_config.py

```diff
@@ -859,7 +858,7 @@ def validate(self) -> None:
         if bool(os.environ.get("RLLIB_ENABLE_RL_MODULE", False)):
             # enable RLModule API and connectors if env variable is set
             # (to be used in unittesting)
-            self._enable_rl_module_api = True
+            self.rl_module(_enable_rl_module_api=True)
```

Comment:
Makes it so that we set exploration_config to {} instead of only setting enable_rl_module_api.